.gitignore

```diff
@@ -11,3 +11,9 @@ vendor
 /captures
 firebase.xml
 google-services.json
+*.patch
```

Comment:
I made these changes because:
 - I use .patch files to revert thinks I just added for a quick test.
 - Already build apk's and build information shouldn't be pushed.